### ISSUE TITLE
[UWP] onAnnotationsChanged not called for every annotation removal

### ISSUE THREAD

**Issue by leoek** [PSPDFKit/react-native#285]
# Details

`onAnnotationsChanged` is not called for every change which happens to the Annotations. The specific annotation which causes issues is a freetext_callout annotation created on android. Does windows change the type when an annotation with unknown type is added? (See the example annotation which I exported from windows now.)

Not sure but this is probably related to: #278 

If any annotation is removed

**Current**

`onAnnotationsChanged` is not called.

**Expected**

`onAnnotationsChanged` is called with the "removed" event and the removed annotations.

**Example Annotation**

```
{
  '0': {
    font: 'Roboto',
    fontColor: '#000000',
    text: '',
    fontSize: 16,
    verticalAlign: 'top',
    createdAt: '2019-08-26T13:11:46Z',
    horizontalAlign: 'left',
    callout: {
      end: [
        402.10614013671875,
        385.60003662109375
      ],
      knee: [
        352.10614013671875,
        385.60003662109375
      ],
      start: [
        302.10614013671875,
        435.60003662109375
      ]
    },
    v: 1,
    creatorName: 'Anr APPB01 Tit APPB01 - ganz langer Titel a. Vorname (APPB01) A. Nachname (alles, APPB01, Alleskönner)',
    updatedAt: '2019-08-26T13:11:46Z',
    pageIndex: 0,
    bbox: [
      272.10614013671875,
      356.10003662109375,
      180,
      109.5
    ],
    type: 'pspdfkit/text',
    opacity: 1,
    name: 'cc9fd071-9f96-4448-b253-fdc4df2776ca'
  }
}
```

# Acceptance Criteria

**Comment by leoek** [comment]
@nickwinder Yeah I was a bit confused because I exected the type to be freetext_callout

**Comment by nickwinder** [comment]
I've just had a look at this issue but I'm unable to reproduce with the annotation provided. 
Is the annotation added as part of a `ImportJson` or `addAnnotation`? 
Note the `callout` information will be dropped from the annotation because this is not supported in UWP.

**Comment by leoek** [comment]
It was imported with `addAnnotation`. What do you mean by "Note the callout information will be dropped from the annotation", is the information just ignored or removed from the annotation?

I will try to reproduce this in our demo application.

**Comment by nickwinder** [comment]
With the annotation referenced, I'm able to add the annotation and remove the annotation and receive events from `onAnnotationsChanged` on for both of these operations. Note: I added contents in the `text` property to ensure it could be seen on the UI for testing.

> What do you mean by "Note the ca...."
Because `callout` is not supported by UWP, it will be dropped from the annotation. `callout` is just a property of an annotation, therefore the annotation in question is a Text annotation with a callout property assigned. In iOS and Android this will be shown, in UWP this will be dropped as it is not supported.

**Comment by steipete** [comment]
I've made a +1 on our feature idea tracker for Callouts on Windows. Goal is to deliver this within 2020, reach annotation feature parity.

**Comment by leoek** [comment]
@nickwinder I will try to reproduce this again. I've got into this situation with an annotation from android on windows which makes it harder to reproduce.

**Comment by leoek** [comment]
>Because callout is not supported by UWP ...
Thanks for the clarification. I think we will disable callout annotations for now as we have not only issues with them on windows (#276)
